ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ContractIdIT.scala

```diff
@@ -30,18 +30,13 @@ import scala.util.{Failure, Success, Try}
 // - Distributed ledger implementations (e.g. Canton) must reject non-suffixed CID
 final class ContractIdIT extends LedgerTestSuite {
   List(
-    TestConfiguration(
-      description = "v0",
-      example = v0Cid,
-      accepted = true,
-      isSupported = features => features.contractIds.v0.isSupported,
-      disabledReason = "V0 contract IDs are not supported",
-    ),
     TestConfiguration(
       description = "v0",
       example = v0Cid,
       accepted = false,
-      isSupported = features => features.contractIds.v0.isNotSupported,
+      // Support for v0 contract ids existed only in sandbox-classic in
+      // SDK 1.18 and older and has been dropped completely.
+      isSupported = _ => true,
       disabledReason = "V0 contract IDs are supported",
```

Comment:
You can just delete these two lines; they have default values.

Reply:
dropped thanks